code/__HELPERS/icons.dm

```diff
@@ -925,11 +925,33 @@ GLOBAL_LIST_EMPTY(friendly_animal_types)
 		return J
 	return 0
 
+//Inefficient pooling/caching way.
+GLOBAL_LIST_INIT(human_dummy_list, list())	//Dummy mob = In use (TRUE/FALSE)
+
+/proc/generate_or_wait_for_human_dummy(slotkey)

```

Comment:
Why not put all this crap in dummy.dm, then you wouldn't have to move the UNTIL macro